sdk/src/main/java/io/opentelemetry/sdk/trace/export/MultiSpanExporter.java

```diff
@@ -67,18 +68,19 @@ public ResultCode export(Collection<SpanData> spans) {
    * @return the result of the operation
    */
   @Override
-  public ResultCode flush() {
-    ResultCode currentResultCode = SUCCESS;
+  public CompletableResultCode flush() {
+    final CompletableResultCode compositeResultCode = new CompletableResultCode();
+    final CountDownLatch completionsToProcess = new CountDownLatch(spanExporters.length);
```

Comment:
is anything waiting on this latch?

Reply:
Nothing needs to wait - See https://github.com/open-telemetry/opentelemetry-java/pull/1428/files#diff-99eb9ce2c0a6fa65d07ccbbee42ba836R107 - when the latch reaches zero, given that there's been a success for the entire composition of result codes, we then succeed the composite.

Reply:
If not waiting on it, may as well use `AtomicInteger` to clarify that it's just for counting.